cfme/base/ui.py

```diff
@@ -323,6 +323,36 @@ def step(self):
             self.prerequisite_view.navigation.select('Settings', 'Configuration')
 
 
+class MySettingsView(BaseLoggedInPage):
+
+    @View.nested
+    class tabs(View):  # noqa
+
+        @View.nested
+        class visual_all(Tab):  # noqa
+            TAB_NAME = "Visual"
+
+        @View.nested
+        class default_views(Tab):  # noqa
+            TAB_NAME = "Default Views"
+
+        @View.nested
+        class default_filter(Tab):  # noqa
+            TAB_NAME = "Default Filters"
+
+        @View.nested
+        class time_profile(Tab):  # noqa
+            TAB_NAME = "Time Profiles"
```

Comment:
Wouldn't it be better to place all widgets in this view?

Reply:
Different tabs have different widgets, so I have placed them separately in different views.
Should I add it here or not?

Reply:
Moreover it would be easier in future if anything needs to be changed in settings related widget, it can be done from settings.py page. Correct me if I am wrong :)